src/app/components/core/control-dispatcher/list-control-dispatcher/acl-list/acl-list.component.ts

```diff
@@ -33,7 +33,23 @@ export class AclListComponent implements OnInit {
     this.controlService.fetchAllACL()
       .subscribe((data: any) => {
         this.ngProgress.done();
-        this.controlAcls = orderBy(data.acls, 'name');
+        this.controlAcls = orderBy(data.acls, 'name')
+        // Uncomment below code for testing of usese on list
+        // this.controlAcls = this.controlAcls.map((item) => {
+        //   item['users'] = [
+        //     {
+        //       "service": "Substation 11",
+        //       "script": "A#6"
```

Comment:
I think these (dummy fixtures) in real are expected to be separate objects; We should probably have type for Service so that we can link to south/north instance details in GUI, otherwise list of services and scripts a better way to avoid duplicates etc.